packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isInImage..st

```diff
@@ -0,0 +1,11 @@
+accessing
+isInImage: aChangeRecord
+	"Compares the source code of aChangeRecord to the code that is currently in the image."
+	| class currentSource changeSource |
+	
+	class := aChangeRecord methodClass.
+	(class notNil and: [class includesSelector: aChangeRecord methodSelector])
+		ifFalse: [^ false].
+	currentSource := (class sourceCodeAt: aChangeRecord methodSelector) asString.
```

Comment:
`methodName` temp var

Reply:
Seems like the correct term for method names in Smalltalk is "selector".
That would also make it more consistent how we retrieve these values:
```smalltalk
class := aChangeRecord methodClass.
selector := aChangeRecord methodSelector.
```